compute/admin_guide/waas/out_of_band.adoc

```diff
@@ -1,5 +1,7 @@
 == Deploying WAAS out of band
 
```

Comment:
Isn't this file outdated? As we improved its content for OOB with VPC Traffic mirroring and moved it under "deploy_waas" directory. So, this file shouldn't be included, what do you suggest?

Reply:
Yes it is, I was not sure why it's still there so I added the comment just to be on the safe side. It's not part of the rendered docs

Reply:
Let's make sure that other changes are properly published and i'll create a separate pull request to remove this file.